Userland/Libraries/LibVideo/VP9/ContextStorage.h

```diff
@@ -168,7 +168,7 @@ class Vector2D {
         VERIFY(height() <= other.height());
         for (u32 row = 0; row < height(); row++) {
             auto other_index = other.index_at(row, 0);
-            AK::TypedTransfer<T>::copy(&m_storage[index_at(row, 0)], &other[other_index], width());
+            memcpy(&m_storage[index_at(row, 0)], &other[other_index], width() * sizeof(T));
```

Comment:
@Zaggy1024  is it expected that all T's for this class will be trivially copyable? If not, I can add the TypedTransfer back in to make sure their copy constructors get called when applicable.

(and if yes, we should document that constraint with a concept or two :P )